stories/feed-input/feed-input.stories.js

```diff
@@ -177,11 +190,11 @@ export const inputGrowth = () => `
                   aria-label="Feed" 
                   aria-required 
                   rows="4">With every line\n"textarea" grows upward\nuntil it reaches\nthe max height</textarea>
-        <button class="fd-button fd-feed-input__submit-button" 
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" 
```

Comment:
To create a button with icon we compose it like that:
```
<button>
    <i class="sap-icon-class"></i>
</button>
```
There is no need to add `sap-icon--feeder-arrow` class to the `button` element.

Reply:
Thanks Michal. I removed the "In Development" tag but fixing all the button code maybe needs another PR?